src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -193,6 +201,19 @@ public function getDefault(): string|bool|int|float|array|null
         return $this->default;
     }
 
+    /**
+     * Returns suggested values for input completion.
+     */
+    public function getSuggestedValues(CompletionInput $input): array
+    {
+        $values = $this->suggestedValues;
+        if ($values instanceof \Closure && \is_array($values = $values($input))) {
+            throw new LogicException(sprintf('Closure for "%s" "%s" must return an array. Got "%s".', $input->getCompletionType(), $input->getCompletionName(), get_debug_type($values)));
```

Comment:
we know the completion type here and above, let's inline the value:
`Closure for option "%s"...`